migration-overview.md

```diff
@@ -12,24 +12,24 @@ aliases: ['/docs-cn/dev/migration-overview/','/docs-cn/dev/data-migration-route'
 
 目前推荐使用以下两种方式将 MySQL 数据迁移到 TiDB。
 
-- [使用 Mydumper 和 TiDB Lightning 迁移全量数据](#使用-mydumper-和-tidb-lightning-迁移全量数据)
+- [使用 Dumpling 和 TiDB Lightning 迁移全量数据](#使用-dumpling-和-tidb-lightning-迁移全量数据)
 - [使用 DM 迁移数据](#使用-dm-迁移数据)
 
-### 使用 Mydumper 和 TiDB Lightning 迁移全量数据
+### 使用 Dumpling 和 TiDB Lightning 迁移全量数据
 
 #### 适合场景
 
 适合 MySQL 全量数据的大小大于 1TB 的场景。该方案只能迁移全量数据，如果需要继续同步增量数据，需要再使用 TiDB Data Migration (DM) 创建增量同步任务。
 
 #### 迁移方法
 
-使用 Mydumper 导出 MySQL 的全量数据，再使用 TiDB Lightning 将全量数据导入到 TiDB，详细信息可参考[从 Mydumper 文件迁移](/migrate-from-mysql-mydumper-files.md)
+使用 Dumpling 导出 MySQL 的全量数据，再使用 TiDB Lightning 将全量数据导入到 TiDB，详细信息可参考[从 Dumpling 文件迁移](/migrate-from-mysql-dumpling-files.md)
```

Comment:
🤔

Reply:
Does anything need to be changed here?

Reply:
https://github.com/pingcap/docs-cn/pull/4857#pullrequestreview-526878360